student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(name, age, parent_permission)
```

Comment:
- [ ] After adding the defaults for name and parent_permission you need to arrange the items in the super method too, to avoid errors.

- [ ] The parent_permission and name use different ways of assignment, unlike what we are used to.

  ```
  super(age, parent_permission: parent_permission, name: name)
  ```